client/lib/sites-list/index.js

```diff
@@ -23,12 +22,14 @@ module.exports = function() {
 					_sites.removeSite( action.site );
 					break;
 				case InvitesActionTypes.INVITE_ACCEPTED:
-					let sites = clone( _sites.get() );
-					if ( sites.length === 0 ) {
-						action.invite.site.primary = true;
+					if ( action.invite.role !== 'follower' ) {
```

Comment:
I thought viewers where users without privileges, @roccotripaldi confirmed to me that they are not users, added `viewers` to the `if` :+1: